index.json

```diff
@@ -16,10 +16,17 @@
   {
     "id": "photoprism",
     "name": "PhotoPrism",
-    "description": "AI-Powered Photos App for the Decentralized Web.",
+    "description": "AI-Powered Photos App for the Decentralized Web. It makes use of the latest technologies to tag and find pictures automatically without getting in your way.",
     "directory": "photoprism",
     "icon": "metadata/icon.png"
   },
+  {
+    "id": "Pi Hole",
```

Comment:
Id doesn't allow spaces, this is reported by your ide as well.
Id's shouldn't have upperCase